web/libs/editor/src/tools/Manager.js

```diff
@@ -123,12 +123,12 @@ class ToolsManager {
 
     if (selected) {
       this.unselectAll();
-      if (tool.setSelected) tool.setSelected(true);
+      tool.setSelected?.(true);
     } else {
       const drawingTool = this.findDrawingTool();
 
       if (drawingTool) return this.selectTool(drawingTool, true);
-      if (tool.setSelected) tool.setSelected(false);
+      this.selectTool(this._default_tool, true);
```

Comment:
This could be cleaned up logically, at first glance I thought it was 2 different calls made but realized that it is just trying to select the most relevant tool. Really minor, just mentioning because the way it read made me double take.

Reply:
thank you, improved this moment!